IaC/pxc.cd/JenkinsStack.yml

```diff
@@ -389,7 +389,7 @@ Resources:
       AvailabilityZone:
         Fn::Select: [ 1, !GetAZs '' ]
       Encrypted: false
-      Size: 200
+      Size: 100
```

Comment:
is it possible to decrease Volume size?

Reply:
It was 100 inside Cloudformation stack in AWS, so I just ported this change not to cause any issue